css/css-fonts/variations/at-font-face-font-matching.html

```diff
@@ -111,6 +111,16 @@
                 }());
             });
         }
+        function createFontFaceRules(fontFaceFamily, descriptorName, expectedMatch, unexpectedMatch) {
+            dynamicStyles.innerHTML =
+                "@font-face { font-family: " + fontFaceFamily + "; src: url('./resources/csstest-weights-100-kerned.ttf'); "+ descriptorName + ": " + expectedMatch   + "; }" +
+                "@font-face { font-family: " + fontFaceFamily + "; src: url('./resources/csstest-weights-200-kerned.ttf'); " + descriptorName + ": " + unexpectedMatch + "; }";
+
+            return Promise.all([
+                load(fontFaceFamily, descriptorName, expectedMatch.split(" ")[0]),
```

Comment:
Actually, the use of `.split(" ")[0]` here is too crude for the case of font-style, as it will result in stripping `oblique <angle1> <angle2>` down to a simple `oblique`, completely losing the custom angle.

Possible fix: pass the `expectedMatch` or `unexpectedMatch` unmodified into `load()`, and within there, process it with something like this:

            if (value.indexOf("deg") > 0) {
              value = "oblique " + value.split(" ")[1];
            } else {
              value = value.split(" ")[0];
            }

That causes a bunch more of the tests to pass reliably in Firefox. There are still a few residual failures, which result from losing the distinction between the expected and unexpected resources because of only considering the range-start here. So for a more complete solution, I think you need to create a pair of spans for each of the rules, and apply the range-start property to one of them and the range-end property to the other.

Taking all that into account, I came up with this for the `load()` function:

        function load(family, name, value) {
            const el1 = document.createElement("span");
            const el2 = document.createElement("span");
            el1.innerText = "A";
            el2.innerText = "A";
            let value1, value2;
            if (value.indexOf("deg") > 0) {
              value1 = "oblique " + value.split(" ")[1];
              value2 = "oblique " + (value.split(" ")[2] || value.split(" ")[1]);
            } else {
              value1 = value.split(" ")[0];
              value2 = value.split(" ")[1] || value1;
            }
            el1.style[name] = value1;
            el2.style[name] = value2;
            document.body.appendChild(el1);
            document.body.appendChild(el2);
            const initialWidth1 = el1.offsetWidth;
            const initialWidth2 = el2.offsetWidth;
            return new Promise((resolve) => {
                el1.style.fontFamily = family;
                el2.style.fontFamily = family;
                (function check() {
                    if (el1.offsetWidth !== initialWidth1 && el2.offsetWidth !== initialWidth2) {
                        el1.remove();
                        el2.remove();
                        resolve();
                    } else {
                        requestAnimationFrame(check);
                    }
                }());
            });
        }

That finally makes the matching tests run reliably for me.